scripts/generate-rtk-apis.ts

```diff
@@ -23,6 +23,25 @@ const config: ConfigFile = {
         'getDashboardByUid',
       ],
     },
+
```

Comment:
Wonder how to make this work for Enterprise. Should we define the same file there with own paths and then somehow merge it with this one?

Reply:
Yeah, that's a valid question. Where would the API clients for enterprise live? Given the legacy api enterprise spec lives in the OSS repo, i think it's fine to generate the clients into OSS repo as well.

Reply:
Yeah, that's valid. The problem is that there's this desire to share at least some of these API clients with plugins, probably through the grafana-runtime package.